content/en/blog/news/2022-gsoc-followup.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Google Summer of Code 2022 ☀️"
+date: 2022-03-12
+draft: false
+description: >
+  Jenkins X GSoC 2022 followup!
+categories: [blog]
+keywords: [Community, 2022]
+slug: "GSoC2022-followup"
+aliases: []
+author: Ankit D Mohapatra
+---
+
+We are very happy to announce that Jenkins X has been selected to participate in the Google Summer of Code (GSoC) 2022!
+
+If you are new to GSoC and want to learn more about it, check out their new [site](https://summerofcode.withgoogle.com/).
+You can find the list of project ideas from Jenkins X [here](/blog/2022/02/20/gsoc2022-ideas/).
+If you are interested in applying to Jenkins X, please check the [timeline](https://developers.google.com/open-source/gsoc/timeline).
+Apply as early as possible.
+
+This post aims to give GSoC contributors insight into the selection process and how to get started.
+We want to be transparent about the entire process.
+
+### How many contributors are we looking for?
+
+We have 3 mentors signed up at the moment and are looking to select maximum 2 applicants.
+
+We want to ensure we have enough time available to mentor individual contributors.
```

Comment:
You could take out this sentence. Again, it's a priority goal to do this for your GSoC org team, not something the applicants should be wondering about.

Reply:
good call